src/App.vue

```diff
@@ -187,6 +200,27 @@ export default class App extends Mixins(
     return process.env.ABOUT_TEXT
   }
 
+  /** The route breadcrumbs list. */
+  get breadcrumbs (): Array<BreadcrumbIF> {
+    const crumbs: Array<BreadcrumbIF> = [
+      {
+        text: 'Name Request',
+        to: { name: Routes.REQUEST }
+      }
+    ]
+
+    // Set base crumbs based on user role
+    // Staff don't want the home landing page and they can't access the Manage Business Dashboard
+    if (this.isRoleStaff) {
+      // If staff, set StaffDashboard as home crumb
+      crumbs.unshift(getStaffDashboardBreadcrumb(sessionStorage.getItem('BUSINESSES_URL')))
+    } else {
+      // For non-staff, set Home crumb
+      crumbs.unshift(getRegistryDashboardBreadcrumb(sessionStorage.getItem('REGISTRY_HOME_URL')))
```

Comment:
I'm using `sessionStorage` here for now as that seems to be what is most commonly used on this project. I can move this to a state getter instead if needed.

Reply:
Since you already use session storage in the resource file, I recommend moving the getItem calls into the methods there instead (which will simplify THIS code and not significantly complicate the resource code).

Reply:
Looks great, thanks.